src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.test.js

```diff
@@ -270,6 +270,47 @@ describe('convertInclude', () => {
     });
   });
 
+  it('should fetch and convert an include block with a propagated renderer_env value', async () => {
```

Comment:
Could we repeat this test for `test` `renderer_env` (maybe use a for loop to parameterize the test).